### ISSUE TITLE
Filters are bugged / Cannot change anything.

### ISSUE THREAD

**Issue by christiangruber** [Vysci/LFG-Bulletin-Board#199]
It's hard for me to add a screenshot of the bug, but as the title says: **I cannot change anything in the app.** 

Whatever I change, settings, filters, channels etc. it's getting reverted to the previous last functional state. I'm also not sure if the filters work because I still see a lot of WTS, GDKP Posts even if they are blacklisted. Do you have any idea how I can reset the add-on?

**Comment by Vysci** [comment]
After you make your changes are you clicking the "X" button or hitting the "Close" button after making changes? 

As far as seeing WTS, GDKP etc..., did you add those words to the "blacklist words" box and did you then also have "Custom" checked under search patterns (along with whatever language)

**Comment by jhaubrich** [comment]
can confirm this is a problem in sod

**Comment by Vysci** [comment]
> can confirm this is a problem in sod.
> 
> To Reproduce:
> 
> 1. Click any setting checkbox (e.g. Lock minimap button position)
> 2. Close config or click away to another config screen
> 3. Open config. See that Lock minimap button position is unchecked.
> 
> I've asked other players if they have this issue. Some do. Some do not.

After you make your changes are you clicking the "X" button or hitting the "Close" button after making changes?

**Comment by christiangruber** [comment]
Yes, interestingly, pressing the close Button instead of X resolved the
problem for me.

**Comment by jhaubrich** [comment]
oh, it does for me too. Pressing "Close" saves the settings. I never click the X or Close. I have always pressed ESC when done with the options window.

**Comment by jhaubrich** [comment]
I made a [bug report for this](https://github.com/Vysci/LFG-Bulletin-Board/issues/200). I'll see if I can fix it.

**Comment by juemrami** [comment]
Using #200 to track this issue instead.
